server/controllers/image.js

```diff
@@ -56,20 +59,24 @@ exports.uploadImages = asyncHandler(async (req, res, next) => {
 // @desc Upload profile photo
 // @access Private
 exports.uploadProfilePhoto = asyncHandler(async (req, res, next) => {
-  const [image] = await helperUploadImages(res, [req.file], req.user);
-
-  const user = await User.findByIdAndUpdate(
-    req.user.id,
-    { profilePhoto: image.id },
-    { new: true }
-  );
+  let user = await User.findById(req.user.id);
```

Comment:
```suggestion
  const user = await User.findById(req.user.id);
```
`const` doesn't mean the variable is immutable, only that its not re-assignable 👍🏼 Unless, you are supposed to have re-assingment of `user = ...` somewhere, we should be using `const`. (you can give [this article](https://dev.to/savagepixie/mutability-and-reassignability-in-javascript-10on) a read if you're unfamiliar with the difference)